### ISSUE TITLE
Bug: incorrect link to the images in the component input

### ISSUE THREAD

**Issue by wwily** [p-foundation/pink-app#38]
**Describe the bug**
The phone and email icons in the input component are not working. The reason for this is an incorrect link to their image files. Although they are displayed correctly on the Live server, the problem occurs when the website is deployed. This is because the Live server can find the root of the project, but when we deploy our site, it doesn't work.

**To Reproduce**
1. Go to `components.html`
2. Scroll down to `input`
3. There should be icons on the gray background in the inputs.
![image](https://github.com/p-foundation/pink-app/assets/126797593/7979ed56-fe56-4dda-b901-b082c50663c5)

**Expected behavior**
If you fix the error, the icons should appear and the inputs should look like this:
![image](https://github.com/p-foundation/pink-app/assets/126797593/1b401495-5d37-4cd7-95d6-d2ddd2113f04)

**Desktop:**
 - OS: **Windows**
 - Browser: **Сhrome**
 - Version: **123.0.6312.59**

**Additional context**
Here is how the code for images looks now:

```css
.input__wrapper_type_phone::after {
    background-image: url(/images/input-phone.svg);
}
```
```css
.input__wrapper_type_email::after {
    background-image: url(/images/input-mail.svg);
}
```

**Quick Instruction for Working on the Issue**
‼️ Before checking out to a new branch, ensure you have the latest updates from the main by running `git pull` command.
- **Please create a new branch**, and name it `fix-links-to-input-icons`
- **Make Changes:** I suggest revising the path to the photo in the `hero` section. Follow the example and correct the mistakes.
```css
.hero {
        background-image: url(../../images/hero-phone-desktop.png);
}
```
- **Test thoughtfully:** go to the [deployed site](https://p-foundation.github.io/pink-app/components.html), open the developer tools, change the path to the icons there, and check if they have appeared.
- **Open a pull request:** reference the specific issue in your PR description using the following format:
`This pull request resolves [#issue_number](link_to_the_issue)`. Replace `issue_number` with the actual issue number and `link_to_the_issue` with the link to the respective issue. For example:

```bash
This pull request resolves [#123](https://github.com/username/repository/issues/123)
```

**Comment by MarPostovik** [comment]
Hey @wwily This issue looks great 👍 . But I have one suggestion. How about giving a hint about the way to solution of the issue instead of providing the exact code the person should use?  You could provide an example of writing the full path to the images in other parts of our project to let the person understand what do you mean. This way other developers could dive deeper into the issue, understand it better, and develop further their skills instead of just copying and pasting the code. :)

**Comment by naumch1k** [comment]
@MarPostovik Great catch! Encouraging people to find their own solutions instead of giving them the answers directly leads to a more fulfilling learning experience 😉

Additionally, @MarPostovik @wwily, do you see any potential issues with this testing approach?
> Test thoughtfully: go to components.html and check if the icons have appeared.

**Comment by MarPostovik** [comment]
@naumch1k Well, I think we can only find out if we have fixed the issue only after we merge it into the main. As on the live server it already works, even with these links. Maybe you have on mind other testing options?

**Comment by naumch1k** [comment]
@MarPostovik, that's exactly right! Do you think we can open our [deployed site](https://p-foundation.github.io/pink-app/components.html) and try updating paths through developers' tools to see if a particular path works?

**Comment by MarPostovik** [comment]
@naumch1k It's a great idea! 
@wwily I think you should include this in your issue description as a way of testing.

**Comment by MarPostovik** [comment]
@wwily thank you for making changes but I think that we can better explain the testing way. 

> Test thoughtfully: go to the [deployed site](https://p-foundation.github.io/pink-app/components.html) and check if the icons have appeared.

Somebody, who will address this issue may think that if he corrects the path to images and then opens the deployed site, he will immediately see the images displayed there. But he won't see them until his changes are merged into the main. But he still can make them appear if he goes to developer tools and tries to change the path to images there.

So let's correct this part of the description like this:

> Test thoughtfully: go to the [deployed site](https://p-foundation.github.io/pink-app/components.html), open the developer tools, change the path to the icons there, and check if they have appeared.

**Comment by wwily** [comment]
Done, thank you for the advice!

**Comment by Woolfs1423** [comment]
.take
